db/migrate/20161118150205_add_user_email_fingerprint.rb

```diff
@@ -0,0 +1,40 @@
+class AddUserEmailFingerprint < ActiveRecord::Migration
+  class User < ActiveRecord::Base
+  end
+
+  def up
+    add_column :users, :email_fingerprint, :string, default: ''
+    add_column :users, :encrypted_email, :text, default: ''
```

Comment:
we need to add an `email_cost` column right? For the same reasons as we did for password and recovery code?